app/controllers/site.rb

```diff
@@ -90,8 +112,43 @@ class Site < Base
 				end
 			end
 
+			namespace "/:site/users" do
+				get "/" do
+					authorize site, :manage_users?
+					users = Tint.db[:users].where(user_id: site.users.map { |u| u[:user_id] }).map do |user|
```

Comment:
I'm a little confused as to what this block is doing, particularly the role bit. Can you explain?